forecast_model.R

```diff
@@ -124,14 +125,14 @@ forecast <- forecast |>
   select(model_id, datetime, reference_datetime, site_id, family, parameter, variable, prediction)
 
 #Visualize forecast.  Is it reasonable?
-#forecast %>%
+#forecast |>
 #  ggplot(aes(x = time, y = prediction, group = ensemble)) +
 #  geom_line() +
 #  facet_grid(variable~site_id, scale ="free")
 
 #Forecast output file name in standards requires for Challenge.
 # csv.gz means that it will be compressed
-file_date <- forecast$reference_datetime[1]
+file_date <- Sys.Date() #forecast$reference_datetime[1]
```

Comment:
date was coming up NA for me so I just penciled this in, maybe not the right choice.

Reply:
Weird that this is an NA.  It should be `forecast$reference_datetime[1]` so that teams know that they using the reference_datetime in the filename